### PR TITLE
Add token to metamask

### PR DESCRIPTION
**Pull request by anxolin**
# Summary

Adds a new button to add to Metamask the VCow token

image

image

# To Test

1. Claim a token
2. Add it to Metamask

### CODE REVIEW

**Comment by elena-zh** [comment]
Hey @anxolin , nice changes!
Some my notes: 
1. I'd rather not to show 'Add to MM' buton when I claim on behalf on an account
on behalf

2. 'Added to MM' is displayed when the token is not yet added/cancelled adding. Can we add some kind of a loader when it is in 'Adding' state and show 'Added' when the asction is successful/show 'Add to MM' when the action is cancelled?
I have to notice, that the same behavior is applicable to the the button on an order placement modal.
3. I have faced an old issue #243 which we decided not to fix: 'Added to MM' button is clickable, calls add to MM flow again

**Review by alfetopito — APPROVED** [review]

File: src/custom/constants/tokens/index.ts [hunk 1]
```diff
@@ -31,28 +31,34 @@ export const ADDRESS_IMAGE_OVERRIDE = {
...
+)
+
+const V_COW_TOKEN_RINKEBY = new Token(
+  SupportedChainId.RINKEBY,
+  V_COW_CONTRACT_ADDRESS[SupportedChainId.RINKEBY] || '',
+  18,
+  'vCOW',
+  'CoW Protocol Virtual Token'
+)
+
```
[the remark this answers is not in the document]
Comment (anxolin) [reply src/custom/constants/tokens/index.ts:57]: sorry for including this in the PR, not a big reason tbh. I needed at first `V_COW_TOKEN_MAINNET` to be declared to use it for something else. Then i was not needed it any more but I just left it like that. 

I think is easier to read though, but  happy to roll that back if u feel is better

**Comment by anxolin** [comment]
> 1. I'd rather not to show 'Add to MM' buton when I claim on behalf on an account

Done! 

> 2. 'Added to MM' is displayed when the token is not yet added/cancelled adding. Can we add some kind of a loader when it is in 'Adding' state and show 'Added' when the asction is successful/show 'Add to MM' when the action is cancelled?
I have to notice, that the same behavior is applicable to the the button on an order placement modal.

Yes, i used the same code. We rely on Metamask API for this. I'm not sure it can be done. Anyways, I just reuse the code as a low hanging fruit. I think it's goodish enough given the big list of things we have


> 3. I have faced an old issue repeatedly asked to add token to MM #243 which we decided not to fix: 'Added to MM' button is clickable, calls add to MM flow again

Same as above. We depend on MM allowing this

**Comment by anxolin** [comment]
Just a picture of it

image

**Comment by W3stside** [comment]
Approved

**Pull request merged**